blocks/certifier.go

```diff
@@ -163,7 +168,7 @@ func (c *Certifier) run() error {
 	for layer := c.layerClock.GetCurrentLayer(); ; layer = layer.Add(1) {
 		select {
 		case <-c.layerClock.AwaitLayer(layer):
-			_ = c.prune()
+			_ = c.genCertificate()
```

Comment:
certificate should be generatable after required weight was received. i don't understand how it makes sense to you